glassfish-tck-runner/pom.template.xml

```diff
@@ -493,7 +493,7 @@
                 <dependency>
                     <groupId>fish.payara.arquillian</groupId>
                     <artifactId>arquillian-payara-server-managed</artifactId>
-                    <version>3.0.alpha3</version>
+                    <version>3.0.alpha6</version>
```

Comment:
can you please move all versions to specific property on the properties section? We know those will change in future releases of the CDI spec

Reply:
done